### PR TITLE
Updated locator for 'parameter_remove' to fix UI tests

### PR DESCRIPTION
**Pull request by sghai**
This will fix two failed tests mentioned in #2584 
`test_remove_domain_parameter`
`test_remove_os_parameter`

### CODE REVIEW

**Comment by sghai** [comment]
```
@Test: Remove a selected domain paramter ... ok

----------------------------------------------------------------------
Ran 1 test in 79.337s

OK

```

**Comment by sghai** [comment]
```
@Test: Remove selected OS parameter ... ok

----------------------------------------------------------------------
Ran 1 test in 57.462s

OK
```

**Comment by oshtaier** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

File: tests/foreman/ui/test_domain.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def test_set_domain_parameter_negative_2(self):
 
     @data(*generate_strings_list(len1=4))
     def test_remove_domain_parameter(self, name):
-        """@Test: Remove a selected domain paramter
+        """@Test: Remove a selected domain parameter
```
Comment (elyezer) [inline tests/foreman/ui/test_domain.py]: :+1:

**Pull request merged**